extensions/ql-vscode/package.json

```diff
@@ -206,6 +206,11 @@
           "default": null,
           "description": "Path to a directory where the CodeQL extension should store query server logs. If empty, the extension stores logs in a temporary workspace folder and deletes the contents after each run."
         },
+        "codeQL.runningQueries.quickEvalCodelens": {
+          "type": "boolean",
+          "default": true,
```

Comment:
Defaulted to `true`, in the assumption that it'll be obvious enough for users to change this in settings.

(If not, I think that GitLens has a "Toggle Code lens" command, so we could do something similar if need be?)

Reply:
I think defaulting to `true` is fine.